patterns/karpenter/main.tf

```diff
@@ -166,6 +157,13 @@ module "eks_blueprints_addons" {
   tags = local.tags
 }
 
+resource "aws_eks_access_entry" "karpenter_node_access_entry" {
+  cluster_name      = module.eks.cluster_name
+  principal_arn     = module.eks_blueprints_addons.karpenter.node_iam_role_arn
+  kubernetes_groups = []
```

Comment:
This line is causing an issue when you try to re-apply terraform

```
aws_eks_access_entry.karpenter_node_access_entry: Modifying... [id=<Karpnter-iam-role-arn>]
╷
│ Error: updating EKS Access Entry (<Karpnter-iam-role-arn>): operation error EKS: UpdateAccessEntry, https response error StatusCode: 400, RequestID: e55f3b9f-b1e0-4c30-bfc8-a1a86b8f024e, InvalidParameterException: The username must not begin with system:
│ 
│   with aws_eks_access_entry.karpenter_node_access_entry,
│   on addons.tf line 230, in resource "aws_eks_access_entry" "karpenter_node_access_entry":
│  230: resource "aws_eks_access_entry" "karpenter_node_access_entry" {
```
Fix is to remove the line

`kubernetes_groups = []` from this resource

Reply:
Yes, the line should not have been there. @vara-bonthu  Do you want to send a PR or want me to do it?